linode/manager
# change: [M3-6465] - Move linodes, k8s & dbs empty landing pages resources to HOC

## Description 📝
Since we created a [new HOC](https://github.com/linode/manager/pull/9078) to consolidate empty pages states, we need to transfer existing ones to use it:
- Linodes
- Kubernetes
- Databases

This PR should not introduce visual changes except where an improvement could be made, or where the styling wasn't consistent with the styles from the `Update empty landing states` epic.

This PR does not need a changeset as there's not change affecting the end user.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-05-08 at 10 45 21 AM](https://user-images.githubusercontent.com/130582365/236855530-d01484ae-40df-4fe0-9a0c-ff545e2e80f8.jpg) | ![Screenshot 2023-05-08 at 10 45 33 AM](https://user-images.githubusercontent.com/130582365/236856025-fc6a3042-3f53-417a-9983-6fad7f57cf23.jpg) |


| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-05-08 at 10 50 07 AM](https://user-images.githubusercontent.com/130582365/236856457-ca90e606-3ec4-4e65-b292-ac603a788df1.jpg) | ![Screenshot 2023-05-08 at 10 50 17 AM](https://user-images.githubusercontent.com/130582365/236856536-0c362722-2d35-4bcb-87fe-c3d9b4b4ead0.jpg) |

## How to test 🧪
1. visit `/linodes` with no linode created
2. visit `/kuberentes` with no k8 created
3. visit `/databases` with no database created

and confirm the pages have no regression (unless it is an improvement) and behave as they should (call to action & resources links destinations)
